SpywareFilter/sections/tracking_servers.txt

```diff
@@ -43,6 +43,7 @@
 ||visilabs.net^$third-party
 !
 ||uncn.jp^$third-party
+||cxense.com^
```

Comment:
Have you checked your rules before adding? We already have this rule, so remove yours. https://github.com/AdguardTeam/AdguardFilters/blob/master/SpywareFilter/sections/tracking_servers.txt#L1237

Reply:
I did, what are you talking about? I added it accidentally then removed it.

Reply:
> I did, what are you talking about? I added it accidentally then removed it.

You added it here https://github.com/AdguardTeam/AdguardFilters/pull/75892/commits/a2041e411526743ac63a3fa4f03f70ff5d500d1a.
Well, please check your changes before `push` to avoid such commits.

Reply:
Okay.